gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -889,4 +1074,40 @@ public static LicenseOption fields(LicenseField... fields) {
    * @throws ComputeException upon failure
    */
   License getLicense(LicenseId license, LicenseOption... options);
+
+  /**
+   * Returns the requested operation or {@code null} if not found.
+   *
+   * @throws ComputeException upon failure
+   */
+  Operation get(OperationId operationId, OperationOption... options);
+
+  /**
+   * Lists the global operations.
+   *
+   * @throws ComputeException upon failure
+   */
+  Page<Operation> listGlobalOperations(OperationListOption... options);
+
+  /**
+   * Lists the operations in the provided region.
```

Comment:
We should update Javadoc to make it clear that it is not a containment based listing but rather type based listing.
Hence, this is a filter for all the regional operations for the specific region rather than all the operations for that region (including all its zones).